setup-scripts/support/logrotate

```diff
@@ -0,0 +1,13 @@
+/home/user/skynet-webportal/docker/data/nginx/logs/*.log {
+    hourly
+    dateext
+    rotate 3650
+    missingok
+    notifempty
+    nocompress
+    create 640 root root
+    sharedscripts
+    postrotate
+        docker exec nginx sh -c '! test -f /usr/local/openresty/nginx/logs/nginx.pid' || docker exec nginx sh -c 'kill -USR1 $(cat /usr/local/openresty/nginx/logs/nginx.pid)'
+    endscript
+}
```

Comment:
add newline at end of file

Reply:
oh shit that file was not supposed to make it there, removing